dev/docker-compose.dev.yml

```diff
@@ -28,7 +27,7 @@ services:
             - auth
     reader:
         build:
-            context: "https://github.com/OpenSlides/openslides-datastore-service.git"
+            context: "https://github.com/OpenSlides/openslides-datastore-service.git#fbe9aca1aa7b29c3ec728876eb530308ee53c165"
```

Comment:
Do you really want to fix the commit? this was previously not the case

Reply:
Right